content/tools/extensions/AzureDevOpsExtension.md

```diff
@@ -1,71 +1,152 @@
 
-### Step two
+<a href="assets/images/AzureDevOps_add_task.png" rel="lightbox" title="Azure DevOps Add Task"><img class="thumbnail" src="assets/images/AzureDevOps_add_task.png"/></a>
 
-* Select a **Service Connection** from the **Contrast Service Connection** field. You can also click on the **Manage** option to go to the **Service connections** settings in your **Project Settings**.
-* Select one of your applications from the **Application** dropdown. This enables more fields for configuring the gate.
+### Step three: Choose a connection and application
 
-<a href="assets/images/AzureDevOps_gate_part1.png" rel="lightbox" title="Azure DevOps Gate Part 1"><img class="thumbnail" src="assets/images/AzureDevOps_gate_part1.png"/></a>
+* Select a **Service Connection** from the **Contrast Service Connection** menu. You can also click on the **Manage** option to go to the **Service connections** settings in your **Project Settings**.
+* Select one of your applications from the **Application** menu.
 
-### Step three
+<a href="assets/images/AzureDevOps_connection_and_app.png" rel="lightbox" title="Azure DevOps Connection And Application"><img class="thumbnail" src="assets/images/AzureDevOps_connection_and_app.png"/></a>
 
-* You can use the **Allowed Status** and **Build Number** fields to filter your results from Contrast. The values set in these fields will be validated against the conditions you configure in the following fields.
+### Step four: Configure the task
+
+* Use the **Allowed Status** and **Build Number** fields to filter your results from Contrast; or, leave them blank, if you don't want to filter. The values set in these fields will be validated against the conditions you configure in the following fields.
 
 <a href="assets/images/AzureDevOps_gate_part2.png" rel="lightbox" title="Azure DevOps Gate Part 2"><img class="thumbnail" src="assets/images/AzureDevOps_gate_part2.png"/></a>
 
-* Proceed to your severity counters, where you must set the maximum number of vulnerabilities allowed per severity. If your selected application has more vulnerabilities than allowed for that severity level, your gate will fail.
+* Proceed to your severity counters, where you must set the maximum number of vulnerabilities allowed per severity. If your selected application has more vulnerabilities than allowed for that severity level, your task will fail.
 
 <a href="assets/images/AzureDevOps_gate_part3.png" rel="lightbox" title="Azure DevOps Gate Part 3"><img class="thumbnail" src="assets/images/AzureDevOps_gate_part3.png"/></a>
 
-> **Note:** Remember that your gates evaluation settings affect all the gates on the current stage for the pre- or post-deployment conditions.
+### Step five: Set job dependency
+
+> For **Build Pipelines** - If you want to prevent the execution of a job if the task fails, you must set the job to depend on the agentless job that includes the Contrast task.
+
```

Comment:
Is step 5 only for build pipelines? Or is this a note in case that's what the reader wants to do?

Reply:
Yes, this is for build pipelines only. The release pipeline doesn't have this option. How should I indicate that it is for Build Pipelines only? Something like 
```
### Step five: Set job dependency (Build Pipelines only)
```
?